pipenv/utils/resolver.py

```diff
@@ -707,15 +702,14 @@ def actually_resolve_deps(
 
 def resolve(cmd, st, project):
     from pipenv.cmdparse import Script
-    from pipenv.vendor.click import echo
 
     c = subprocess_run(Script.parse(cmd).cmd_args, block=False, env=os.environ.copy())
     is_verbose = project.s.is_verbose()
-    err = ""
+    err_ = ""
```

Comment:
Can we name this `errors` or something? -- `err_` isn't a great name

Reply:
yeah we can.